app/src/main/java/org/simple/clinic/patient/businessid/BusinessId.kt

```diff
@@ -85,6 +89,17 @@ data class BusinessId(
 
       @VisibleForTesting(otherwise = VisibleForTesting.NONE)
       fun values() = TypeAdapter.knownMappings.keys
+
+      fun forIdentifierType(
+          type: IdentifierType
+      ): Optional<MetaDataVersion> {
+        return Optional.ofNullable(when (type) {
```

Comment:
Should we extract out the `when` block into a separate variable for readability purposes?

Reply:
Yes, I can do that.